### DISCUSSION TITLE
5.0 tooltip hover 时的辅助线如何配置样式？

### DISCUSSION THREAD

**Discussion by dreamerblue** [antvis/G2#5263] [Q&A] [answered]
- **G2 Version**: 5.0.13
- **Platform**: macOS
- **Mini Showcase(like  screenshots)**: ![](https://mdn.alipayobjects.com/huamei_qa8qxu/afts/img/A*r95yTqow_1EAAAAAAAAAAAAADmJ7AQ/original)
- **CodePen Link**:

如官方文档，对折线图的 tooltip，在使用了自带主题 classicDark 后，这条辅助线亦然是深灰色的，很难辨别。在官方折线图示例中把主题改为 classicDark 就可以复现。

现在想要配置它的颜色，但文档上目前看只有 title 和 items 可以配置，请问想要配置 hover 的这条辅助线应该用哪个选项？

**Comment by pearmini** [comment, marked as answer]
按照 `crosshairs${style}` 的形式

```js
chart.interaction('tooltip', {
  crosshairsStroke: 'red',
  crosshairsLineWidth: 10,
});
```

**Reply by dreamerblue** [reply]
满足了需求，谢谢！
